Formula/packetq.rb

```diff
@@ -0,0 +1,26 @@
+class Packetq < Formula
+  desc "SQL-frontend to PCAP-files"
+  homepage "https://www.dns-oarc.net/tools/packetq"
+  url "https://github.com/DNS-OARC/PacketQ/archive/v1.3.1.tar.gz"
```

Comment:
This does not appear to be the release tarball, but rather the tarball created as an archive of the git tag. The text of the release seems to indicate that https://www.dns-oarc.net/files/packetq/packetq-1.3.1.tar.gz is the actual release tarball.